scripts/globals/abilities/super_jump.lua

```diff
@@ -14,7 +14,13 @@ ability_object.onAbilityCheck = function(player, target, ability)
 end
 
 ability_object.onUseAbility = function(player, target, ability)
+    local wyvern = player:getPet()
+
     xi.job_utils.dragoon.useSuperJump(player, target, ability)
+
+    if (wyvern ~= nil and player:getPetID() == xi.pet.id.WYVERN and wyvern:getHP() > 0) then
+        wyvern:useJobAbility(652, wyvern)
```

Comment:
What ability is this?

You can make it more obvious with:

```lua
wyvern:useJobAbility(xi.jobAbility.SUPER_CLIMB, wyvern)
```

From `scripts/globals/ability.lua`